### PR TITLE
Rasa X annotations

### PR DESCRIPTION
**Pull request by melindaloubser1**

### CODE REVIEW

File: data/nlu.md [hunk 1]
```diff
@@ -205,6 +206,7 @@
 - transfer money
 - I want to transfer $100 to John
 - i want to transfer $400 to Akela
+- who can I transfer money to?
```
Comment (melindaloubser1) [inline data/nlu.md:209]:
```suggestion
```

File: data/nlu.md [hunk 2]
```diff
@@ -213,6 +215,9 @@
 - Show the recipient list
 - Show me people I can send money to
 - Who is a valid payee
+- who can I pay?
+- who can I transfer money?
+- who's in my recipient list?
```
Comment (melindaloubser1) [inline data/nlu.md:220]:
```suggestion
- who's in my recipient list?
- who can I transfer money to?
```

**Comment by b-quachtran** [comment]
Can you add a synonym for gringots -> gringotts? The slot validation for credit_card only works using an exact match (ignoring casing) to the dict key.

**Comment by melindaloubser1** [comment]
odd - I added the synonyms in Rasa X but I guess they didn't get added here?

**Comment by melindaloubser1** [comment]
@b-quachtran added now

**Pull request merged**